File: templates/templates/FSharp.EpoxyHello.Avalonia/EpoxyHello.UI/ViewModels/ItemViewModel.fs
////////////////////////////////////////////////////////////////////////////
//
// Epoxy template source code.
// Write your own copyright and note.
// (You can use https://github.com/rubicon-oss/LicenseHeaderManager)
//
////////////////////////////////////////////////////////////////////////////

namespace EpoxyHello.ViewModels

open Epoxy
open Avalonia.Media.Imaging

[<Sealed; ViewModel>]
type ItemViewModel() =

    member val Title : string = "" with get, set
    member val Image : Bitmap = null with get, set
    member val Score : int = 0 with get, set

File: templates/templates/EpoxyHello.Avalonia/EpoxyHello.UI/ViewModels/ItemViewModel.cs
////////////////////////////////////////////////////////////////////////////
//
// Epoxy template source code.
// Write your own copyright and note.
// (You can use https://github.com/rubicon-oss/LicenseHeaderManager)
//
////////////////////////////////////////////////////////////////////////////

using Epoxy;

using Avalonia.Media.Imaging;

namespace EpoxyHello.ViewModels
{
    [ViewModel]
    public sealed class ItemViewModel
    {
        public string? Title { get; set; }

        public Bitmap? Image { get; set; }

        public int Score { get; set; }
    }
}
